app/graphql/mutations/password_resets/reset.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Mutations
+  module PasswordResets
+    class Reset < BaseMutation
+      graphql_name 'ResetPassword'
+      description 'Reset password for user and log in'
+
+      argument :new_password, String, required: true
+      argument :token, String, required: true
+
+      type Types::Payloads::LoginUserType
+
+      def resolve(new_password:, token:)
+        password_reset = PasswordReset.where('expire_at > ?', DateTime.now).find_by(token:)
```

Comment:
I guess it should be: `PasswordReset.where('expire_at < ?', DateTime.now).find_by(token:)` Probably you also miss test case for this.

Reply:
Done !